### PR TITLE
[post] Symfony Controller param clean way

### PR DESCRIPTION
**Pull request by TomasVotruba**
## Todo

- [ ] add answer + link to stackoverflow

### CODE REVIEW

File: source/_posts/2018/2018-01-21-how-to-get-parameter-in-symfony-controller-the-clean-way.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+    {
+        $this->paymentService = $paymentService;
+    }
+
+	public function registerAction(): void
+	{
+        $bankAccount = $this->container->getParameter('bankAccount');
+        
+        $this->paymentService->payAmountToAccount(1000, $bankAccount);
+	}
```
Comment (carusogabriel) [inline source/_posts/2018/2018-01-21-how-to-get-parameter-in-symfony-controller-the-clean-way.md]: The indentation in this block of code should be fixed :wrench:

File: source/_posts/2018/2018-01-21-how-to-get-parameter-in-symfony-controller-the-clean-way.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+<?php declare(strict_types=1);
+
+final class LectureController extends SymfonyController
+{
+    /**
+     * @var PaymentService
+     */
+    private $paymentService;
+    
+	public function __construct(PaymentService $paymentService)
```
Comment (carusogabriel) [inline source/_posts/2018/2018-01-21-how-to-get-parameter-in-symfony-controller-the-clean-way.md]: The indentation in this block of code should be fixed :wrench:

File: source/_posts/2018/2018-01-21-how-to-get-parameter-in-symfony-controller-the-clean-way.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+     * @var string 
+     */
+    private $bankAccount;
+    
+    /**
+     * @var PaymentService
+     */
+    private $paymentService;
+    
+	public function __construct(string $bankAccount, PaymentService $paymentService)
```
Comment (carusogabriel) [inline source/_posts/2018/2018-01-21-how-to-get-parameter-in-symfony-controller-the-clean-way.md]: The indentation in this block of code should be fixed :wrench:

**Comment by TomasVotruba** [comment]
Thank you, indentation fixed!

**Pull request merged**